config/locales/rosetta/ja.json

```diff
@@ -66,36 +66,36 @@
-      "volunteer_portal.admin.tab.organizations.edit": "Xdxt",
-      "volunteer_portal.admin.tab.organizations.delete": "Dxlxtx",
-      "volunteer_portal.admin.tab.organizations.name": "Nxmx",
-      "volunteer_portal.admin.tab.organizations.location": "Lxcxtxxn",
-      "volunteer_portal.admin.tab.organizations.actions": "Xctxxns",
-      "volunteer_portal.admin.tab.organizations.delete.cancel": "Cxncxl",
-      "volunteer_portal.admin.tab.organizations.delete.delete": "Dxlxtx",
-      "volunteer_portal.admin.tab.organizations.add_organization": "Xdd Xrgxnxzxtxxn",
-      "volunteer_portal.admin.tab.organizations.delete_organization": "Dxlxtx Xrgxnxzxtxxn",
-      "volunteer_portal.admin.tab.organization.delete.confirmation": "Xrx yxx sxrx yxx wxnt tx dxlxtx {{organization}}?",
-      "volunteer_portal.admin.tab.users.edit": "Xdxt",
-      "volunteer_portal.admin.tab.users.delete": "Dxlxtx",
-      "volunteer_portal.admin.tab.users.name": "Nxmx",
-      "volunteer_portal.admin.tab.users.email": "Xmxxl",
-      "volunteer_portal.admin.tab.users.group": "Grxxp",
-      "volunteer_portal.admin.tab.users.actions": "Xctxxns",
-      "volunteer_portal.admin.tab.users.delete.cancel": "Cxncxl",
-      "volunteer_portal.admin.tab.users.delete.delete": "Dxlxtx",
-      "volunteer_portal.admin.tab.users.delete_user": "Dxlxtx Xsxr",
+      "volunteer_portal.admin.tab.offices_edit": "XdXt",
+      "volunteer_portal.admin.tab.offices_delete": "Dxlxtx",
+      "volunteer_portal.admin.tab.offices_name": "Nxmx",
+      "volunteer_portal.admin.tab.offices_timezone": "Txmxzxnx",
+      "volunteer_portal.admin.tab.offices_actions": "Xctxxns",
+      "volunteer_portal.admin.tab.offices_delete.cancel": "Cxncxl",
+      "volunteer_portal.admin.tab.offices_delete.delete": "Dxlxtx",
+      "volunteer_portal.admin.tab.offices_add_office": "Xdd Xffxcx",
+      "volunteer_portal.admin.tab.offices_delete.delete_office": "Dxlxtx Xffxcx",
+      "volunteer_portal.admin.tab.offices_delete.confirmation": "Xrx yxx sxrx yxx wxnt tx dxlxtx {{office}}?",
+      "volunteer_portal.admin.tab.organizations_edit": "Xdxt",
+      "volunteer_portal.admin.tab.organizations_delete": "Dxlxtx",
+      "volunteer_portal.admin.tab.organizations_name": "Nxmx",
+      "volunteer_portal.admin.tab.organizations_location": "Lxcxtxxn",
+      "volunteer_portal.admin.tab.organizations_actions": "Xctxxns",
+      "volunteer_portal.admin.tab.organizations_delete.cancel": "Cxncxl",
+      "volunteer_portal.admin.tab.organizations_delete.delete": "Dxlxtx",
+      "volunteer_portal.admin.tab.organizations_add_organization": "Xdd Xrgxnxzxtxxn",
+      "volunteer_portal.admin.tab.organizations_delete_organization": "Dxlxtx Xrgxnxzxtxxn",
+      "volunteer_portal.admin.tab.organization.delete_confirmation": "Xrx yxx sxrx yxx wxnt tx dxlxtx {{organization}}?",
+      "volunteer_portal.admin.tab.userse_edit": "Xdxt",
```

Comment:
did the bot do a typo 😂 

can we also fix this to `users_edit`? 🙇

Reply:
Ah, that was my bad 🤣